src/style/icomoon/style.css

```diff
@@ -0,0 +1,192 @@
+@font-face {
```

Comment:
As mentioned on slack, we already have an icon package with Semantic UI so lets not import a whole other library into the app

All you have to do is import the icon with `import { Icon } from 'semantic-ui-react'`  then use it like this: `<Icon name='facebook' size='mini' />`  which means we don't have to import a whole icon package